backend/api/go_http_client/pipeline_client/pipeline_service/pipeline_service_client.go

```diff
@@ -213,7 +213,7 @@ func (a *Client) GetPipelineVersion(params *GetPipelineVersionParams, authInfo r
 }
 
 /*
-GetPipelineVersionTemplate get pipeline version template API
+GetPipelineVersionTemplate returns a single y a m l template that contains the description parameters and metadata associated with the given pipeline version
```

Comment:
addressed in proto file and I used "Returns a YAML template that contains the specified pipeline version's description, parameters and metadata." instead, because description is a field in the yaml template.

Reply:
Also, please review in the proto file since the comments in go_http_client folder is auto-generated and doesn't allow manual editing.